modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -13,36 +12,45 @@ def initialize(a_server = nil, a_ttl = nil)
     end
 
     def create_a_record(fqdn, ip)
-      if found = dns_find(fqdn)
-        raise(Proxy::Dns::Collision, "#{fqdn} is already used by #{found}") if found != ip
+      conflict = a_record_conflicts(fqdn, ip) #returns -1, 0, 1
+      if conflict >= 0
```

Comment:
Yea, this was just for the example above, to save me some writing. Commit coming right up!